include/zephyr/sys/arch_interface.h

```diff
@@ -1098,17 +1104,25 @@ void arch_timing_stop(void);
 /**
  * @brief Return timing counter.
  *
+ * @parblock
```

Comment:
I am not sure the added parblocks are necessary and making any difference?

Reply:
These keeps the note sections separated or else both would be merged together as one big note block.

Reply:
Oh you're quite right. I must have done something wrong when I thought I was testing how it looked on /latest and had the impression it was looking just the same